README.md

```diff
@@ -46,7 +46,7 @@ Non-boolean options:
 |error_class| StandardError | supported | supported | Change validation errors from `Committee::ValidationError`). |
 |prefix| String | supported | supported | Mounts the middleware to respond at a configured prefix. (e.g. prefix is '/v1' and request path is '/v1/test' use '/test' definition). |
 |schema_path| String | supported | supported | Defines the location of the schema file to use for validation. |
-|error_handler| Proc Object | supported | supported | A proc which will be called when error occurs. Take an Error instance as first argument. |
+|error_handler| Proc Object | supported | supported | A proc which will be called when error occurs. Take an Error instance as first argument, and request.env as second argument. |
```

Comment:
Please change error_handler description in ResponseValidation document too

Reply:
It would be better if there was an example of error handling.

Reply:
Sure. I'm going to add example